pkgs/development/python-modules/pycritty/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder, pyyaml }:
+
+buildPythonPackage rec {
+  pname = "pycritty";
+  version = "0.3.5";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1lrmd4a1ps3h9z0pndfjfrd2qa7v3abd6np75fd2q2ffsqv7ar6x";
+  };
+
+  # The package does not include any tests to run
+  doCheck = false;
+
+  propagatedBuildInputs = [ pyyaml ];
+
+  postPatch = ''
+    # remove custom install
+    substituteInPlace setup.py \
+      --replace "'install': PostInstallHook," ""
+  '';
+
+  meta = with lib; {
+    description = "A CLI tool for changing your alacritty configuration on the fly.";
+    homepage    = "https://github.com/antoniosarosi/pycritty";
+    license     = licenses.mit;
+    maintainers = with maintainers; [ jperras ];
+  };
+
+}
```

```suggestion
  };
}
```

I would like to suggest to use `nixpkgs-fmt` for the initial formatting. Otherwise if another person is providing an update then the style will be updated anyway.

---

@fabaff I've run `nixpkgs-fmt` on the PR. Thanks for pointing that out!

---

nixpkgs-fmt does not enforce whitespace style.